docs/faststore/docs/managing-performance/managing-lazy-loading-sections-on-plp.mdx

```diff
@@ -0,0 +1,172 @@
+
+While lazy loading improves performance, it can harm SEO. Search engine crawlers often do not execute JavaScript or wait for dynamically loaded content. As a result, critical sections like product grids on category pages may not be indexed. For instance, if a user accesses the "gym clothes" collection, its product listing might not appear in search results if it's not included in the initial HTML.
+
+In this guide, learn how to optimize SEO by controlling lazy loading for sections in FastStore.
+
+> ⚠ This feature may increase initial page load times, potentially impacting performance and user experience. Use it only when necessary.
+
+## Before you begin
+
+Make sure the `@faststore/cli` package is updated to the latest version. If you need to update it, follow the instructions on [Updating the '@fastore/cli' package version](https://developers.vtex.com/docs/guides/faststore/project-structure-updating-the-cli-package-version).
+
+## Understand how SEO metadata is determined
+
+FastStore follows a specific priority order when generating SEO metadata for PLP pages:
+
+<details>
+<summary>Title</summary>
+
+```mermaid
+graph LR
+A[Checks the Catalog for PLP's title] --> B{Is there a title in the Catalog?};
+B -- Yes --> C[Uses the title from the Catalog];
+B -- No --> D[Checks the Catalog for PLP's name];
+D --> E{Is there a name in the Catalog?};
+E -- Yes --> F[Uses the name from the Catalog];
+E -- No --> G[Checks 'discovery.config.js' file];
+G --> H[Uses general SEO title];
+C --> I[Displays Title];
+F --> I;
+H --> I;
+```
+
+</details>
+
+<details>
+<summary>Title template</summary>
+
+```mermaid
+graph LR
+A[Checks for a template configured in 'discovery.config.js' in the 'seo' section] --> B{Is there a title template?};
+B -- Yes --> C[Displays Title using the template];
+B -- No --> D[Uses the generic title template from the 'seo' object in 'discovery.config.js'.];
+D --> C;
+```
+
+</details>
+
+<details>
+<summary>Description template</summary>
+
+```mermaid
+graph LR
+A[Checks the Catalog for the PLP's description] --> B{Is there a description in the Catalog?};
+B -- Yes --> C[Uses the description from the Catalog];
+B -- No --> D[Checks the PLP description template in the 'plp' property of 'discovery.config.js'];
+D --> E{Is there a PLP description template?};
+E -- Yes --> F[Uses the PLP description template using the PLP title in it];
+E -- No --> G[Uses the generic description template in the 'seo' object of 'discovery.config.js'];
+C --> H[Displays Description];
+F --> H;
+G --> H;
+```
+
+</details>
+
+## Instructions
+
+### Step 1: Configuring SEO for Product Listing Pages (PLP)
```

Comment:
de repente isso deveria estar dentro de before you begin. achei que ficou meio deslocado esse conteúdo aqui